src/pages/academy/components/_subscribe.js

```diff
@@ -186,6 +186,7 @@ const Subscribe = () => {
             .catch(() => {
                 setSubmitStatus(false)
             })
+        user_country && console.log(user_country)
```

Comment:
We can remove this line add 'user_country' in the fallback of useEffect

Reply:
Whoops 😅